servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/DefaultExecutor.java

```diff
@@ -249,11 +254,27 @@ public Cancellable schedule(final Runnable task, final long delay, final TimeUni
             // When using the global scheduler, offload timer ticks to the user specified Executor since user code
             // executed on the timer tick can block.
             ScheduledFuture<?> future = GLOBAL_SINGLE_THREADED_SCHEDULED_EXECUTOR.schedule(
```

Comment:
if `offloadExecutor` is a `ScheduledExecutorService` instance then perhaps schedule with it directly?

Reply:
Fair enough. This class appears to be used only for non-`ScheduledExecutorServices` instances anyway as elsewhere in `Executors` the `from(ScheduledExecutorService scheduled)` method handles executors that support scheduling.